report_builder/tests.py

```diff
@@ -551,6 +559,7 @@ def test_groupby_id(self):
 
         data = '"data":[[1,"John","Doe",3],[3,"Donald","King",6],[2,"Maria","Smith",2],["TOTALS","","",""],["","","",11.0]]'
 
+        print(response.data)
```

Comment:
if you need more debugging info i would just have a custom error message if the test failed with extra information